src/js/components/ToggleButtonGroup/ToggleButtonGroup.js

```diff
@@ -0,0 +1,205 @@
+import React, { useCallback, useContext, useState, useRef } from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { Keyboard } from '../Keyboard';
+import { normalizeColor, edgeStyle } from '../../utils';
+import { ToggleButtonGroupPropTypes } from './propTypes';
+
+const StyledButton = styled(Button)`
+  ${(props) =>
+    edgeStyle(
+      'padding',
+      props.theme.toggleButtonGroup.button.pad,
+      false,
+      undefined,
+      props.theme,
+    )}
+  border-radius: ${(props) => props.theme.global.control.border.radius};
+  border: none;
+  &:hover {
+    box-shadow: none;
+    background: ${(props) =>
+      normalizeColor(
+        props.theme.global.colors['background-contrast'],
+        props.theme,
+      )};
+    color: ${(props) =>
+      normalizeColor(props.theme.toggleButtonGroup.button.color, props.theme)};
+  }
+`;
```

Comment:
if we remove the extra &hover it will just be 
But i can move to another file just put it here since its on the shorter end.
```
const StyledButton = styled(Button)`
  ${(props) =>
    edgeStyle(
      'padding',
      props.theme.toggleButtonGroup.button.pad,
      false,
      undefined,
      props.theme,
    )}
  border-radius: ${(props) => props.theme.global.control.border.radius};
  border: none;
`;

```

Reply:
done